plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorSmokeTest.java

```diff
@@ -30,4 +36,31 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                 return super.hasBehavior(connectorBehavior);
         }
     }
+
+    @Override
+    public void testRenameSchema()
```

Comment:
> Enable SUPPORTS_RENAME_SCHEMA in this class and override the test in TestClickHouseConnectorSmokeTest (not here). e.g.

I think the following code is enough because: 1. we can't get `TestingClickHouseServer` in class `TestClickHouseConnectorSmokeTest` 2. the compatible tests already exists in `TestClickHouseConnectorTest`.
In this case, when we execute `ALTER SCHEMA ... RENAME` on an older version of `clickhouse-server`, we only need to check that the error message is as expected.

```
    @Test
    @Override
    public void testRenameSchema()
    {
        // Override because the default database engine in version < v20.10.2.20-stable doesn't allow renaming schemas
        assertThatThrownBy(super::testRenameSchema).hasMessageMatching("ClickHouse exception, code: 48,.* Ordinary: RENAME DATABASE is not supported .*\\n");
    }
```

Reply:
> we can't get TestingClickHouseServer in class TestClickHouseConnectorSmokeTest

We can fix that. See for example https://github.com/trinodb/trino/blob/51577a6ac91e9076abbcd1296054d934ce9e78b6/plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java#L47

Reply:
@tangjiangling Let's keep exisiting way - I didn't notice that you were talking about the override in smoke test only. We will have a `*LatestSmokeTest` which wouldn't need the override and hence be covered.